addon/index.js

```diff
@@ -23,6 +24,7 @@ export { findElementWithAssert } from './-private/extend/find-element-with-asser
 export { buildSelector, getContext } from './-private/helpers';
 
 export default {
+  alias,
```

Comment:
Can you add `alias` to the exports test? https://github.com/san650/ember-cli-page-object/blob/217d647ec34ad8b1686d824d16a2253fbcd5e22b/tests/unit/addon-exports-test.js